llvm/test/tools/llvm-readobj/ELF/string-dump.test

```diff
@@ -23,7 +23,9 @@
 # RUN: llvm-readelf -hp1 -p2 %t | cmp %t.readelf.out -
 # RUN: llvm-readelf -hp 1 -p.not_null_terminated %t | cmp %t.readelf.out -
 
-# CHECK:      String dump of section '.strings':
+# HEADER:     LoadName:
```

Comment:
Same as above.

Reply:
llvm-readobj -x .a -x .b prints two blank lines before the first Hex dump of section.
We have to test `LoadName:` to skip the first blank line (not due to hex/string dump).